src/main/kotlin/blackjack/domain/Cards.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain
+
+data class Cards(private val cards: MutableList<Card> = mutableListOf()) : List<Card> by cards {
```

Comment:
```suggestion
data class Cards(private val cards: List<Card> = listOf()) : List<Card> by cards {
```
내부에서 add 혹은 remove를 하지 않기 때문에, List 타입으로 변경해도 될 것 같아요

Reply:
Cards는 내부적으로 add를 하고 있어서 변경은 하지 못했지만
Players 변경했습니다!
f9681a6